### DISCUSSION TITLE
Provide lock reason on mobile when locking issues or prs

### DISCUSSION THREAD

**Discussion by jonchurch** [community/community#122361] [Mobile]
### Select Topic Area

Product Feedback

### Body

I'd like to use the "lock with reason" feature from the website on github mobile.

We've had a lot of [spam PRs](https://github.com/expressjs/express/pulls?q=is%3Apr+is%3Aclosed+update+readme.md) on express, and I mostly triage that on mobile. When we lock these PRs we usually give the reason "spam" to make it clear.

That isn't an option when locking from mobile, there is no option to provide a lock reason.

I'd like to be able to lock w/ a reason on mobile

**Comment by ghost** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> I'd like to use the "lock with reason" feature from the website on github mobile.
> 
> 
> 
> We've had a lot of [spam PRs](https://github.com/expressjs/express/pulls?q=is%3Apr+is%3Aclosed+update+readme.md) on express, and I mostly triage that on mobile. When we lock these PRs we usually give the reason "spam" to make it clear.
> 
> 
> 
> That isn't an option when locking from mobile, there is no option to provide a lock reason.
> 
> 
> 
> I'd like to be able to lock w/ a reason on mobile
